### PR TITLE
Develop apd sprint1 fix

### PR DESCRIPTION
**Pull request by jleong01**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/templates/coursepage/.content.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     jcr:description="Taylors University Course Page Template"
     jcr:primaryType="cq:Template"
     jcr:title="Taylors University Course Page Template"
-    allowedPaths="[/content/taylorsuniversity(/.*)?]"
+    allowedPaths="[/content/taylorsuniversity/.*/.*(/.*)?,/content/tu/.*/.*(/.*)?,/content/tc/.*/.*(/.*)?]"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/templates/coursepage/.content.xml]: Please change the ranking as per the latest TDD.

Comment (bsinghal1) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/templates/coursepage/.content.xml]: Also, add allwedParents to it as per the latest TDD

Comment (bsinghal1) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/templates/coursepage/.content.xml]: Also change the allowedPaths as per the latest TDD

**Comment by jleong01** [comment]
Raising new pull request.

**Comment by jleong01** [comment]
Raising new pu

**Pull request closed without merging**